tests/tailsort/loadandsave/tests.csv

```diff
@@ -0,0 +1,11 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+before,0,0,q,.ts.hdbdir:`:/home/edalzell/cloudTorQ/TorQ/tests/tailsort/loadandsave/hdb,1,,"set test HDB path"
+before,0,0,q,.ts.taildir:`:/home/edalzell/cloudTorQ/TorQ/tests/tailsort/loadandsave,1,,"set test tailDB path"
+before,0,0,q,pt:2022.08.19,1,,"set partition for loadandsave function"
+before,0,0,q,procname:`tailer1`tailer2,1,,"set procnames for loadandsave function to locate tailDBs"
+run,0,0,q,loadandsave[pt;] each procname,1,,"run loadandsave function for both tailDBs"
+true,0,0,q,(`$string pt) in key .ts.hdbdir,1,,"check date partition was created in HDB"
+true,0,0,q,`quote in key ` sv (.ts.hdbdir;`$string pt),1,,"check quote table is present in HDB partition"
+true,0,0,q,`trade in key ` sv (.ts.hdbdir;`$string pt),1,,"check trade table is present in HDB partition"
+true,0,0,q,not (`$string pt) in key ` sv (.ts.taildir;`wdb1),1,,"check date partition was deleted in tailDB"
+true,0,0,q,not (`$string pt) in key ` sv (.ts.taildir;`wdb2),1,,"check date partition was deleted in tailDB"
```

Comment:
I think these should be `tailer1` and `tailer2` instead of `wdb1` and `wdb2`?

```suggestion
true,0,0,q,not (`$string pt) in key ` sv (.ts.taildir;`tailer1),1,,"check date partition was deleted in tailDB"
true,0,0,q,not (`$string pt) in key ` sv (.ts.taildir;`tailer2),1,,"check date partition was deleted in tailDB"
```

Reply:
yep you're right - must've missed that when we split the wdb and tailers to different procs